### ISSUE TITLE
Incorrect tree shaking in release 3.7.2 causing entire mui icons to be included

### ISSUE THREAD

**Issue by spareelectronics** [gregnb/mui-datatables#1603]
In issue 3.7.2 incorrect tree shaking, probably in in src/components/ExpandButton.js line 3,
is causing entire material-ui/icons module to be included.

I am using Webpack 5.10.0, so that may also play a factor.

I am thinking the issue could be fixed by either:
1) going back to the safe approach of importing like:
_import Remove from '@material-ui/icons/Remove';_

or ensure tree shaking is conducted

<img width="997" alt="Screen Shot 2020-12-11 at 1 14 26 AM" src="https://user-images.githubusercontent.com/22668900/101871381-766a4d80-3b51-11eb-8592-4a639a153713.png">

## Expected Behavior

Entire icons module should not be included

## Current Behavior

Entire icons module is included

## Steps to Reproduce (for bugs)

1.  Build project all else equal with release 3.7.2, make note of bundle size
2.  uninstall 3.7.2, and install 3.7.1, it should be significantly smaller

## Your Environment

To verify that the problem was with mui-datatables I used verbose mode on webpack along with bundle analyzer, to narrow down where the images were being imported in. After finding that my vendor module with the icons, only occurred on pages where I used mui-datatables, I followed the steps in "steps to reproduce" and found that fixed the extra icons.

If anyone else is having this problem, I could probably submit a pull request with a fix.

| Tech         | Version |
|--------------|---------|
| Material-UI  |   4.11.2      |
| MUI-datatables  |   Used both 3.7.2 and 3.7.1       |
| React        |    16.14     |
| browser      |    Chrome     |
| Webpack         |     5.10    |

**Comment by wdh2100** [comment]
@spareelectronics 

thanks report!

I changed the settings through the document below, but this seems to affect tree shaking.
https://material-ui.com/guides/minimizing-bundle-size/

It can be changed enough if it is proven.


```js
 //AS-iS 
import { Add as AddIcon } from '@material-ui/icons';
import { Tabs } from '@material-ui/core';
//                                 ^^^^ 1st or top-level

//TO-BE
import AddIcon from '@material-ui/icons/Add';
import Tabs from '@material-ui/core/Tabs';
//                                  ^^^^ 2nd level
``

**Comment by wdh2100** [comment]
If I have time, I'll measure it with this.

https://www.npmjs.com/package/webpack-bundle-analyzer

**Comment by iacovlev-pavel** [comment]
Have the same issue,
with `import MUIDataTable from 'mui-datatables'` 838 kB
without `import MUIDataTable from 'mui-datatables'` 4.42 kB

Checked with webpack-bundle-analyzer, looks like the entire @material-ui/icons package is included.

Using latest next.js

**Comment by spareelectronics** [comment]
The mui docs link encourages library authors to use the path import instead of the top level import. I'll do some more testing with the latest version of mui datatables, and see if I can reduce the bundle size. I'll share images from bundle anaylzer, here once I'm done.  Here's the where it says to use path import: 
![Screenshot_20201222-110408_Chrome](https://user-images.githubusercontent.com/22668900/102908573-a95feb80-4445-11eb-8e92-1942f35e5e9c.jpg)

**Comment by bmlkc** [comment]
confirming. had to fork and fix imports. tree shaking doesn't work with material-ui when you use imports like:

```
import { Add as AddIcon } from '@material-ui/icons';
import { Tabs } from '@material-ui/core';
```

**Comment by wdh2100** [comment]
Fixed 3.7.4
